easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2020b-Perl-5.32.0.eb

```diff
@@ -0,0 +1,72 @@
+##
+# This file is an EasyBuild recipy as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# https://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Updated to use gompi-2020b toolchain:
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'HH-suite'
+version = '3.3.0'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://github.com/soedinglab/hh-suite'
+description = """The HH-suite is an open-source software package 
+for sensitive protein sequence searching based on the pairwise 
+alignment of hidden Markov models (HMMs)."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+
+source_urls = ['https://github.com/soedinglab/hh-suite/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dd67f7f3bf601e48c9c0bc4cf1fbe3b946f787a808bde765e9436a48d27b0964']
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('Perl', '5.32.0'),
+]
+
+separate_build_dir = True
```

Comment:
This is the default now, so you can just remove this line

Reply:
Ok. I will remove that.